src/web_app/chrome_app/manifest.json

```diff
@@ -5,7 +5,7 @@
   "manifest_version": 2,
   "app": {
     "background": {
-      "scripts": ["js/background.js"]
+      "scripts": ["js/apprtc.debug.js", "js/background.js"]
```

Comment:
I thought about doing this, but we would be shipping the same code twice in the chrome app package, because appwindow uses apprtc.debug.js, because it uses the same code as the apprtc on the web.